external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -5,7 +5,7 @@
     <!-- Given that xunit packages bring with them part of the framework, we need to specify a runtime in order to get the assets
          This RID value doesn't really matter, since the assets we are copying are not RID specific, so defaulting to Windows here
     -->
-    <NugetRuntimeIdentifier>win7-x64</NugetRuntimeIdentifier>
+    <NugetRuntimeIdentifier>win7-$(ArchGroup)</NugetRuntimeIdentifier>
```

Comment:
> If the RID does matter then we should also update the comment.

I'll update the comment.

> Why isn't sni coming in from the runtime.depproj? 

This is a file that has to go into the TestILCInputFolder and the runtime.depproj specifically disables the binplacing to that folder since we don't want the rest of the runtime to be there, so if we wanted that project to deploy sni.dll to that folder we would have to add extra logic there, which is why I went with piggy-backing on the xunit.depproj which does binplace to TestILCInputFolder.